src/main/java/gregtech/common/blocks/BlockWireCoil.java

```diff
@@ -74,6 +74,11 @@ public boolean canCreatureSpawn(@Nonnull IBlockState state, @Nonnull IBlockAcces
         return false;
     }
 
+    @Override
+    protected boolean isBloomEnabled(CoilType value) {
+        return ConfigHolder.client.casingsActiveEmissiveTextures;
+    }
```

Comment:
To match the current description of this config. I think the fusion casing should also override this method and depend on the config value.